dali/operators/reader/loader/video/frames_decoder_test.cc

```diff
@@ -115,6 +115,16 @@ class FramesDecoderTestBase : public VideoTestBase {
 
 class FramesDecoderTest_CpuOnlyTests : public FramesDecoderTestBase {
  public:
+  // due to difference in CPU postprocessing on different CPUs eps is 10
+  void RunSequentialTest(FramesDecoder &decoder, TestVideo &ground_truth, double eps = 10.) {
```

Comment:
Have you actually seen errors that big?

Reply:
See https://github.com/NVIDIA/DALI/blob/main/dali/operators/reader/loader/video/video_test_base.h#L59.
I'm just restoring the original behavior. I think the test video frames are so distinct that even eps = 10 still gives us a guarantee we are decoding what we suppose to.